guides/v2.2/frontend-dev-guide/layouts/layout-overview.md

```diff
@@ -10,36 +10,34 @@ github_link: frontend-dev-guide/layouts/layout-overview.md
 
-<h2>Introduction</h2>
+## Introduction
 In Magento, the basic components of page design are layouts, containers, and blocks. A *layout* represents the structure of a web page (1). *Containers* represent the placeholders within that web page structure (2). And *blocks* represent the UI controls or components within the container placeholders (3). These terms are illustrated and defined below.
 
-<img src="{{ site.baseurl }}/common/images/layouts_block_containers_defn21.png"/>
+![web page sample layout][layout]
 
 (1) *Layouts* provide the structures for web pages using an XML file that identifies all the containers and blocks composing the page. The details of layout XML files are described later in this section.
 
 (2) *Containers* assign content structure to a page using container tags within a layout XML file. A container has no additional content except the content of included elements. Examples of containers include the header, left column, main column, and footer.
 
 (3) *Blocks* render the UI elements on a page using block tags within a layout XML file. Blocks use templates to generate the {% glossarytooltip a2aff425-07dd-4bd6-9671-29b7edefa871 %}HTML{% endglossarytooltip %} to insert into its parent structural block. Examples of blocks include a {% glossarytooltip 50e49338-1e6c-4473-8527-9e401d67ea2b %}category{% endglossarytooltip %} list, a mini cart, product tags, and product listing.
 
-## Basic layouts 
+## Basic layouts
 
-The basic view of all Magento {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} pages in defined in two page configuration layout files located in the Magento_Theme module: 
+The basic view of all Magento {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %} pages in defined in two page configuration layout files located in the Magento_Theme module:
 
-* `<Magento_Theme_module_dir>/view/frontend/layout/default.xml`: defines the page layout. 
-* `<Magento_Theme_module_dir>/view/frontend/layout/default_head_blocks.xml`: defines the scripts, images, and meta data included in pages' `<head>` section. 
+* `<Magento_Theme_module_dir>/view/frontend/layout/default.xml`: defines the page layout.
+* `<Magento_Theme_module_dir>/view/frontend/layout/default_head_blocks.xml`: defines the scripts, images, and meta data included in pages' `<head>` section.
 
 These basic page configuration layouts are extended in other Magento modules and in Magento themes.
 
-You can also [extend]({{ page.baseurl }}/frontend-dev-guide/layouts/layout-extend.html) or [override]({{ page.baseurl }}/frontend-dev-guide/layouts/layout-override.html) these files in your custom theme. 
+You can also [extend]({{ page.baseurl }}/frontend-dev-guide/layouts/layout-extend.html) or [override]({{ page.baseurl }}/frontend-dev-guide/layouts/layout-override.html) these files in your custom theme.
```

Comment:
use link definitions instead of direct hyperlinks

Reply:
fixed